tests/playwright/fixtures/asset-paths.html

```diff
@@ -1,44 +0,0 @@
-<!doctype html>
-<html>
-  <head>
-    <script src="https://cdn.tailwindcss.com"></script>
```

Comment:
@tevanoff I blew away the tailwind script from all of the fixtures. I'm assuming it was just used to make the styling a little better, but let me know if we were wanting to make sure external scripts with styles get archived and I can make a test for it.

Reply:
@skitterm yeah it was there to implicitly test that styles from external files were being inlined and applied properly, so it could use an explicit test.

Reply:
I added a test for this.